task/04-arrays-tasks.js

```diff
@@ -536,7 +613,8 @@ function selectMany(arr, childrenSelector) {
  *   [[[ 1, 2, 3]]], [ 0, 0, 1 ]      => 2        (arr[0][0][1])
  */
 function getElementByIndexes(arr, indexes) {
-    throw new Error('Not implemented');
+   let place=indexes.map(elem=>{return `[${elem}]`}).join("");
```

Comment:
arrow functions doesn't have to have a body e.g.` let place = indexes.map(elem => '[${elem}]').join("");`